services/server/deps.go

```diff
@@ -62,9 +62,16 @@ type alertmanagerService interface {
 	healthChecker
 }
 
-// prometheusAlertingRules is a subset of methods of prometheus.AlertingRules used by this package.
+// vmAlertService is a subset of methods of vmalert.Service used by this package.
+// We use it instead of real type to avoid dependency cycle.
+type vmAlertService interface {
+	RequestConfigurationUpdate()
+	healthChecker
+}
+
+// vmAlertAlertingRules is a subset of methods of vmalert.ExternalAlertingRules used by this package.
 // We use it instead of real type for testing and to avoid dependency cycle.
-type prometheusAlertingRules interface {
+type vmAlertAlertingRules interface {
```

Comment:
Let's call them "external rules" everywhere (names, comments, etc): they are not only "alerting" anymore

Reply:
Done